src/main/java/com/poliscrypts/api/controller/CompanyController.java

```diff
@@ -0,0 +1,39 @@
+package com.poliscrypts.api.controller;
+
+
+import com.poliscrypts.api.entity.Company;
+import com.poliscrypts.api.model.ExtendedGenericPojoResponse;
+import com.poliscrypts.api.model.GenericPojoResponse;
+import com.poliscrypts.api.service.CompanyService;
+import io.swagger.annotations.Api;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api")
+@Api(tags = "Company")
+public class CompanyController {
+
+    @Autowired
+    CompanyService companyService;
+
+    @GetMapping("/companies")
```

Comment:
don't expose your PK's, use and expose a uuid field instead (you can keep your id as Integer and add uuid field)